openapi_spec_validator/__init__.py

```diff
@@ -50,6 +57,18 @@
     resolver_handlers=default_handlers,
 )
 
+# v3.1 spec
+schema_v31, schema_v31_url = get_openapi_schema('3.1')
+openapi_v31_validator_factory = Draft202012JSONSpecValidatorFactory(
+    schema_v31, schema_v31_url,
+    resolver_handlers=default_handlers,
+)
+openapi_v31_spec_validator = SpecValidator(
```

Comment:
maybe we should create `default` and `v3` aliases to point to the  newest 3.1 version and  name v3.0 explicitly?

Reply:
Should we use `latest` instead of `default`? I think renaming `v3` to `v30` and having an alias for `v3` to `v31` sound pretty good.

Reply:
@p1c2u I adapted the shortcuts, please check if this is what you expected